js/widgets/fixedToolbar.js

```diff
@@ -43,10 +43,11 @@
 			transition: "slide", //can be none, fade, slide (slide maps to slideup or slidedown)
 			fullscreen: false,
 			tapToggle: true,
-			tapToggleBlacklist: "a, button, input, select, textarea, .ui-header-fixed, .ui-footer-fixed, .ui-flipswitch, .ui-popup, .ui-panel, .ui-panel-dismiss-open",
+			tapToggleBlacklist: "a, button, input, select, textarea, .ui-popup, " +
+				".ui-toolbar-header-fixed, .ui-toolbar-footer-fixed, .ui-flipswitch, " +
+				".ui-panel, .ui-panel-dismiss-open",
 			hideDuringFocus: "input, textarea, select",
 			updatePagePadding: true,
-			trackPersistentToolbars: true,
 
 			// Browser detection! Weeee, here we go...
```

Comment:
@gabrielschulhof @cgack Im thinking we should just drop all these fixed support stuff right now since it does not apply any more?